### ISSUE TITLE
Native Base fails on flowtype 0.37.0

### ISSUE THREAD

**Issue by blinky9** [GeekyAnts/NativeBase#633]
flowtype (version 0.37.0) gives error for native-base 2.0.6.
here is some of errors:

node_modules/native-base/src/backward/Widgets/Button.js:8
  8: import variables from '../../theme/variables';
                           ^^^^^^^^^^^^^^^^^^^^^^^ ../../theme/variables. Required module not found

node_modules/native-base/src/backward/Widgets/Button.js:89
 89:         ref={(c) => { this._root = c; }}
                                ^^^^^ property `_root`. Property not found in
 89:         ref={(c) => { this._root = c; }}
                           ^^^^ Button

node_modules/native-base/src/backward/Widgets/Card.js:32
 32:       <View ref={c => this._root = c} {...this.props} >
                                ^^^^^ property `_root`. Property not found in
 32:       <View ref={c => this._root = c} {...this.props} >
                           ^^^^ Card

node_modules/native-base/src/backward/Widgets/CardItem.js:195
195:         ref={(c) => { this._root = c; }} {...this.prepareRootProps()}
                                ^^^^^ property `_root`. Property not found in
195:         ref={(c) => { this._root = c; }} {...this.prepareRootProps()}
                           ^^^^ CardItem1

....

**Comment by ericvan76** [comment]
I don't think native-base is flow-typed, you may need to ignore it from your .flowconfig

**Comment by designorant** [comment]
It seems to be, to some extend at least, judging by the `/* @flow */` comments.

I was getting 133 errors with `native-base@2.0.13` against `flow-bin@0.38.0` on `react-native@0.42` so just added the following to `.flowconfig`:
```
[ignore]
.*/node_modules/native-base/.*
.*/node_modules/react-native-easy-grid/.*
.*/node_modules/react-native-keyboard-aware-scroll-view/.*
```

**Comment by hugomosh** [comment]
ignore and add a mapper like [#275982533](https://github.com/react-community/react-navigation/issues/81#issuecomment-275982533) to avoid **"Required module not found"**

```
[ignore]
.*/node_modules/native-base/.*
.*/node_modules/react-native-easy-grid/.*
.*/node_modules/react-native-keyboard-aware-scroll-view/.*
.*/node_modules/react-navigation/.*

[options]
module.name_mapper='^react-navigation$' -> 'emptyObject'
module.name_mapper='^native-base$' -> 'emptyObject'
module.name_mapper='^react-native-easy-grid$' -> 'emptyObject'
module.name_mapper='^react-native-keyboard-aware-scroll-view$' -> 'emptyObject'

```

**Comment by SupriyaKalghatgi** [comment]
Check commit [4663af0](https://github.com/GeekyAnts/NativeBase/commit/4663af052e6e4852d3c37289503f84201ba04c26)

**Comment by SupriyaKalghatgi** [comment]
Fixed with [v2.5.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.5.2)
